### PR TITLE
feat(SDK-avax-p-lib): add avax-p-lib module

### PR DESCRIPTION
**Pull request by sskim16**
add avax-p-lib module decoupled from account lib.  Avaxp is a utxo based
coin but uses features of account lib.

Ticket: STLX-15980

### CODE REVIEW

File: modules/avax-p-lib/package.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@bitgo/avax-p-lib",
```
Comment (mmcshinsky-bitgo) [inline modules/avax-p-lib/package.json:2]: You'll need to add this package to the root `tsconfig.packages.json` file as well.

Comment (sskim16) [reply modules/avax-p-lib/package.json]: updated!

File: modules/bitgo/src/v2/coins/tavaxp.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { BitGo } from '../../bitgo';
+import { BaseCoin } from '../baseCoin';
+import { AvaxP } from './avaxp';
+import { BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+
+export class TavaxP extends AvaxP {
```
Comment (shiksb) [inline modules/bitgo/src/v2/coins/tavaxp.ts:6]:
```suggestion
export class TAvaxP extends AvaxP {
```

Comment (sskim16) [reply modules/bitgo/src/v2/coins/tavaxp.ts:6]: I think we should follow the syntax of avaxc and keep things consistent.  TavaxC is how its written.

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**